packages/internal/test-util/src/lib/object-driver/object-driver-root.module.ts

```diff
@@ -1,40 +1,22 @@
-import { NgModule, Optional, SkipSelf } from '@angular/core';
+import { inject, InjectFlags, NgModule } from '@angular/core';
 
-import {
-  LumberjackLogDriverConfig,
-  lumberjackLogDriverConfigToken,
-  lumberjackLogDriverToken,
-} from '@ngworker/lumberjack';
+import { lumberjackLogDriverToken } from '@ngworker/lumberjack';
 
-import { objectDriverConfigToken } from './object-driver-config.token';
-import { ObjectDriverConfig } from './object-driver.config';
 import { ObjectDriver } from './object.driver';
-import { ObjectService } from './object.service';
-
-export function objectDriverFactory(
-  logDriverConfig: LumberjackLogDriverConfig,
-  objectDriverConfig: ObjectDriverConfig,
-  objectService: ObjectService
-): ObjectDriver {
-  const baseConfig = { ...logDriverConfig, identifier: ObjectDriver.driverIdentifier };
-  const fullConfig = { ...baseConfig, ...objectDriverConfig };
-
-  return new ObjectDriver(fullConfig, objectService);
-}
 
 @NgModule({
   providers: [
     {
       provide: lumberjackLogDriverToken,
-      useFactory: objectDriverFactory,
-      deps: [lumberjackLogDriverConfigToken, objectDriverConfigToken, ObjectService],
+      useClass: ObjectDriver,
       multi: true,
     },
   ],
 })
 export class ObjectDriverRootModule {
-  constructor(@Optional() @SkipSelf() maybeNgModuleFromParentInjector?: ObjectDriverRootModule) {
-    if (maybeNgModuleFromParentInjector) {
+  maybeNgModuleFromParentInjector = inject(ObjectDriverRootModule, InjectFlags.SkipSelf | InjectFlags.Optional);
```

Comment:
nit: separate property and constructor with a blank line.

Reply:
issue: mark as private.
```suggestion
  private maybeNgModuleFromParentInjector = inject(ObjectDriverRootModule, InjectFlags.SkipSelf | InjectFlags.Optional);
```